src/sys/event.rs

```diff
@@ -22,13 +22,12 @@ pub struct KEvent {
           target_os = "dragonfly", target_os = "macos",
           target_os = "ios"))]
 type type_of_udata = *mut libc::c_void;
-#[cfg(any(target_os = "openbsd", target_os = "freebsd",
-          target_os = "dragonfly", target_os = "macos",
-          target_os = "ios"))]
+#[cfg(any(target_os = "freebsd", target_os = "dragonfly",
```

Comment:
Please alphabetize the order of the OSes since you're already modifying this attribute.